src/app/dataset-block/metadata-block/components/tooltip-icon/tooltip-icon.component.scss

```diff
@@ -1,26 +0,0 @@
-.tooltip-class {
```

Comment:
Please remove this file
And don't forget to run stylelint along with lint  :)

I have plans to add a pre-commit hook with lint + stylelint, for safe commits :D

Reply:
Ok, removed